src/Core/Utilities/LoggerFactoryExtensions.cs

```diff
@@ -70,6 +73,52 @@ bool inclusionPredicate(LogEvent e)
                     .Enrich.FromLogContext()
                     .Enrich.WithProperty("Project", globalSettings.ProjectName);
             }
+            else if (CoreHelpers.SettingHasValue(globalSettings?.Syslog.Destination))
+            {
+                // appending sitename to project name to allow eaiser identification in syslog.
+                var appName = $"{globalSettings.SiteName}-{globalSettings.ProjectName}";
+                if (globalSettings.Syslog.Destination.Equals("local", StringComparison.OrdinalIgnoreCase))
+                {
+                    config.WriteTo.LocalSyslog(appName);
+                }
+                else if (Uri.TryCreate(globalSettings.Syslog.Destination,UriKind.Absolute, out var syslogAddress))
+                {
+                    // Syslog's standard port is 514 (both UDP and TCP). TLS does not have a standard port, so assume 514.
+                    int port = syslogAddress.Port >= 0
+                        ? syslogAddress.Port
+                        : 514;
+
+                    if (syslogAddress.Scheme.Equals("udp"))
+                    {
+                        config.WriteTo.UdpSyslog(syslogAddress.Host, port, appName);
+                    }
+                    else if (syslogAddress.Scheme.Equals("tcp"))
+                    {
+                        config.WriteTo.TcpSyslog(syslogAddress.Host, port, appName);
+                    }
+                    else if (syslogAddress.Scheme.Equals("tls"))
+                    {
+                        // TLS v1.1, v1.2 and v1.3 are explicitly selected (leaving out TLS v1.0)
+                        const SslProtocols protocols = SslProtocols.Tls11 | SslProtocols.Tls12 | SslProtocols.Tls13;
+
+                        if (CoreHelpers.SettingHasValue(globalSettings.Syslog.CertificateThumbprint))
+                        {
+                            config.WriteTo.TcpSyslog(syslogAddress.Host, port, appName,
+                                secureProtocols: protocols,
+                                certProvider: new CertificateStoreProvider(StoreName.My, StoreLocation.CurrentUser, 
```

Comment:
Is a certProvider always required? Could it just be a cert signed by a trusted CA?

Reply:
To clarify, for TCP connections, `certProvider` is not required. 

When `secureProtocols` is assigned a value other than `SslProtocols.None`, `certProvider` must be supplied with a `ICertificateProvider`, that will fetch a `X509Certificate2` from either:
- the current user's `my` certificate store, 
- or a specified file. 

`ICertificateProvider` does not supply a `X509Chain`, which would contain a CA, as that process is left up to .NET to resolve. 

As [`certValidationCallback`](https://docs.microsoft.com/en-us/dotnet/api/system.net.security.remotecertificatevalidationcallback?view=netcore-3.1) has not been supplied, it is assumed the certificate used is signed by a CA (already stored on the host, or in the certificate store).